src/header/navigation.rs

```diff
@@ -16,6 +16,28 @@ pub fn Header() -> impl IntoView {
             href: "/transactions".to_string(),
             text: "Transactions".to_string(),
             icon: NavIcon::Transactions,
+            sub_entries: Some(vec![
+                NavEntry {
+                    href: "/transactions".to_string(),
+                    text: "Transactions".to_string(),
+                    icon: NavIcon::Transactions,
+                    ..Default::default()
+                },
+                NavEntry {
+                    href: "/transactions/token-trxs".to_string(),
```

Comment:
Regarding this: `/transactions/token-trxs`

Since the URL is already underneath `/transactions`, I would just do `/transactions/token` instead of `/transactions/token-trxs`. I know my suggestion might read odd so feel free to disagree.

Reply:
Yes that works